src/cmd-buildupload

```diff
@@ -181,7 +187,13 @@ def s3_copy(src, bucket, key, max_age, acl, extra_args={}, dry_run=False):
         'ACL': acl
     }
     upload_args.update(extra_args)
-    s3 = boto3.client('s3')
```

Comment:
Why not just skip creating the client if it's a dry run?

Reply:
Hmm, I think what's happening here is that `buildextend-vmware` isn't putting the artifact in the right place, so the code is trying to check if the artifact already exists in the bucket. Does `buildextend-vmware` work locally for you?

Reply:
Duh...yeah, over thought that one. Fixed and pushed.